lib/ansible/utils/module_docs_fragments/aws.py

```diff
@@ -63,6 +65,11 @@ class ModuleDocFragment(object):
     default: null
     aliases: []
     version_added: "1.6"
+  region:
```

Comment:
I think this was the reason for the distinct `ec2` and `aws` docs fragments originally. We should probably move to one or the other.

Reply:
Do you think I should I remove the ec2 doc fragment and anywhere it is being used? I don't understand why they were separate. I could also remove this from my PR and edit modules to use the ec2 doc fragment that aren't doing so and need region documented.

Reply:
Loads of non EC2 modules need region. Some AWS services don't (IAM, Cloudfront, Route53). Not sure how best to approach that. Maybe remove ec2 and replace with a region fragment or something?